src/utils/actions.js

```diff
@@ -5,6 +5,9 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
+export const USER_ID = 'Id';
```

Comment:
Should not be in actions.js which is for redux actions.

Reply:
done